osa/scripts/gain_selection.py

```diff
@@ -123,27 +125,47 @@ def apply_gain_selection(date: str, output_basedir: Path = None):
         
         for file in r0_files:
             output_file = output_dir / file.name
-            file.link_to(output_file)
+            sp.run(["cp", file, output_dir])
+
+def run_sacct_j(job) -> StringIO:
+    """Run sacct to obtain the job information."""
+    if shutil.which("sacct") is None:
+        log.warning("No job info available since sacct command is not available")
+        return StringIO()
+
+    sacct_cmd = [
+        "sacct",
+        "-n",
+        "--parsable2",
+        "--delimiter=,",
+        "--units=G",
+        "-o",
+        ",".join(FORMAT_SLURM),
+        "-j",
+        job,
```

Comment:
Now I see that here you are passing the job id. I guess then that `starttime` is not needed?

Reply:
Yes, it is not needed as I am passing the job id